clients/venice-client/src/main/java/com/linkedin/venice/fastclient/factory/ClientFactory.java

```diff
@@ -50,18 +44,12 @@ public static <K, V extends SpecificRecord> AvroSpecificStoreClient<K, V> getAnd
 
   private static StoreMetadata constructStoreMetadataReader(ClientConfig clientConfig) {
     switch (clientConfig.getStoreMetadataFetchMode()) {
```

Comment:
Can we inline this function? And get rid of `getStoreMetadataFetchMode` completely?